src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -45,8 +45,8 @@ public CommandResult execute(Model model) throws CommandException {
         if (model.isFilteredPersonListEmpty()) {
             // View blank
             model.setViewingPerson(null);
-        } else {
-            // Show first employee
+        } else if (personToDelete.equals(model.getViewingPerson().get())) {
```

Comment:
Just a suggestion, but since there is actually a method `Person.isSamePerson()`, maybe using that would be clearer?

Reply:
Thanks, for some reason did not realise that method existed.